src/input/mod.rs

```diff
@@ -333,21 +333,18 @@ impl Hit for axis::Key {
     fn hit(
         &self,
         input: &Input,
-    ) -> bool {
-        input
-            .delta
-            .keys_hit
-            .iter()
-            .filter(|&&k| k == self.pos || k == self.neg)
-            .count() > 0
+    ) > bool {
+        let pos_hit = input.state.keys_pressed.contains(&self.pos);
+        let neg_hit = input.state.keys_pressed.contains(&self.neg);
+        pos_hit.or(neg_hit).is_some()
     }
 }
 
 impl Hit for axis::Raw {
     fn hit(
         &self,
         input: &Input,
-    ) -> bool {
+    ) > bool {
```

Comment:
?

Reply:
Huh, that's weird. I probably pressed 'x' by mistake before pushing.